Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift

```diff
@@ -54,9 +54,13 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
         self.closeButton.setTitle(closeButtonTitle, for: .normal)
     }
 
-    init(title: String? = nil, actions: [[PhotonActionSheetItem]], closeButtonTitle: String = Strings.CloseButtonTitle, style presentationStyle: UIModalPresentationStyle) {
+    init(title: String? = nil, actions: [[PhotonActionSheetItem]], closeButtonTitle: String = Strings.CloseButtonTitle, style presentationStyle: UIModalPresentationStyle? = nil) {
```

Comment:
Make this non-optional again please. The `nil == .centered` I find confusing. Or make the default .centered (`, style presentationStyle: UIModalPresentationStyle = .centered) `if that is what was needed here.

Reply:
Talked about this offline. There's a difference between `UIModalPresentationStyle` and `PresentationStyle` and this is the easiest way to get the `PresentationStyle` set to `.centered` without breaking other callers.